src/App.tsx

```diff
@@ -1,24 +1,276 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import classNames from 'classnames';
+import {
+  addTodo,
+  changeTodo,
+  getTodos,
+  removeTodo,
+} from './api/todos';
+
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+import { ErrorsType } from './types/ErrorsType';
+
+import { filterTodos } from './utils/helpers/filterBySort';
+
 import { UserWarning } from './UserWarning';
+import { HeaderTodo } from './components/HeaderTodo';
+import { TodoList } from './components/TodoList';
+import { TodoFooter } from './components/TodoFooter';
 
-const USER_ID = 0;
+const USER_ID = 7036;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [filterType, setSortType] = useState<FilterType>(FilterType.ALL);
+  const [errorType, setErrorType] = useState<ErrorsType>(ErrorsType.EMPTY);
+  const [isHiddenError, setIsHiddenError] = useState(true);
+  const [loadingTodosIds, setLoadingTodosIds] = useState<number[]>([]);
+
+  const handleError = useCallback((typeOfError: ErrorsType) => {
+    setErrorType(typeOfError);
+    setIsHiddenError(false);
+
+    setTimeout(() => {
+      setIsHiddenError(true);
+      // setErrorType(ErrorsType.NONE);
```

Comment:
Why did you comment this code?

Reply:
Reset error type as well, better to keep correct state values even if you using a separate state for controlling error messages on the UI